lib/open_food_network/orders_and_fulfillments_report.rb

```diff
@@ -89,5 +99,13 @@ def order_permissions
     def report_line_items
       @report_line_items ||= Reports::LineItems.new(order_permissions, options)
     end
+
+    def report_variant_overrides
+      @report_variant_overrides ||=
+        Reports::VariantOverrides.new(
+          line_items: order_permissions.visible_line_items,
+          distributor_ids: report_line_items.orders.result.pluck('DISTINCT distributor_id'),
```

Comment:
Not super happy with this approach, but I couldn't think of a better one. Any ideas?

I had an alternative implementation where the `Reports::VariantOverrides` class was only responsible for loading variant overrides for a single distributor at a time, which made it easier to just inject the relevant distributor each time a new one was encountered, but it obviously resulted in more queries. See 2a843bf33. Let me know which you think is preferable.

Reply:
Lets go with this version :+1: